packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -71,7 +71,7 @@ Requires: %{name}-tesla(fabricmanager)
 %package open-gpu-%{tesla_major}
 Summary: NVIDIA %{tesla_major} Open GPU driver
 Version: %{tesla_ver}
-License: MIT OR GPL-2.0-only
+License: MIT OR (MIT AND GPLv2)
```

Comment:
I don't think the MIT part is accurate given this statement in the license:
> However, when linked together to form a Linux kernel module, the resulting Linux kernel module is dual licensed as MIT/GPLv2.

We are definitely linking it together to form a Linux kernel module here, and the license should reflect that only the dual licensed form applies.